### PR TITLE
ID Localization for Setup - Container Runtime.

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR is addressing one translation page as mentioned in here #22296

### CODE REVIEW

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+{{< /note >}}
+
+Kamu harus menjalankan semua perintah dalam panduan ini sebagai `root`. Sebagai contoh, awali perintah 
+dengan `sudo`, atau masuk sebagai `root` dan kemudian baru menjalankan perintah sebagai pengguna `root`.
+
+### _Driver_ cgroup
+
+Ketika systemd dipilih sebagai sistem init untuk sebuah distribusi Linux, proses init menghasilkan
+dan menggunakan grup kontrol root (`cgroup`) dan proses ini akan bertindak sebagai manajer cgroup. Systemd memiliki integrasi yang ketat
+dengan cgroup dan akan mengalokasikan cgroups untuk setiap proses. Anda dapat mengonfigurasi
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:39]: nit: use kamu
```suggestion
dengan cgroup dan akan mengalokasikan cgroups untuk setiap proses. Kamu dapat mengonfigurasi
```

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+
+{{< /note >}}
+
+Kamu harus menjalankan semua perintah dalam panduan ini sebagai `root`. Sebagai contoh, awali perintah 
+dengan `sudo`, atau masuk sebagai `root` dan kemudian baru menjalankan perintah sebagai pengguna `root`.
+
+### _Driver_ cgroup
+
+Ketika systemd dipilih sebagai sistem init untuk sebuah distribusi Linux, proses init menghasilkan
+dan menggunakan grup kontrol root (`cgroup`) dan proses ini akan bertindak sebagai manajer cgroup. Systemd memiliki integrasi yang ketat
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:38]: should we use _control group_ since it reflects `cgroup` or grup kontrol make sense? in some other places I found you use _control group_

let's discuss and keep them consistent for this entire page

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+dengan cgroup dan akan mengalokasikan cgroups untuk setiap proses. Anda dapat mengonfigurasi
+_runtime_ Container dan kubelet untuk menggunakan `cgroupfs`. Menggunakan `cgroupfs` bersama dengan systemd berarti
+akan ada dua manajer cgroup yang berbeda.
+
+_Control group_ digunakan untuk membatasi sumber daya yang dialokasikan untuk proses.
+Sebuah manajer cgroup tunggal akan menyederhanakan pandangan tentang sumber daya apa yang sedang dialokasikan
+dan secara bawaan (_default_) akan memiliki pandangan yang lebih konsisten tentang sumber daya yang tersedia dan yang sedang digunakan. Ketika kita punya memiliki 
+dua manajer kita akan memiliki dua pandangan tentang sumber daya tersebut. Kita telah melihat kasus di lapangan
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:46]: the sentence doesn't make sense, maybe rephrase to
```suggestion
dan secara bawaan (_default_) akan memiliki pandangan yang lebih konsisten tentang sumber daya yang tersedia dan yang sedang digunakan. Ketika kita memiliki 
dua manajer, maka kita pun akan memiliki dua pandangan berbeda tentang sumber daya tersebut. Kita telah melihat kasus di lapangan
```

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+di mana Node yang dikonfigurasi menggunakan `cgroupfs` untuk kubelet dan Docker, dan `systemd`
+untuk semua sisa proses yang berjalan pada Node maka Node tersebut akan menjadi tidak stabil dibawah tekanan sumber daya.
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:48]: nit: add comma
```suggestion
untuk semua sisa proses yang berjalan pada Node, maka Node tersebut akan menjadi tidak stabil di bawah tekanan sumber daya.
```

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+
+Mengubah aturan sedemikian rupa sehingga _runtime_ Container dan kubelet kamu menggunakan `systemd` sebagai driver cgroup
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:50]: nit: italicize
```suggestion
Mengubah aturan sedemikian rupa sehingga _runtime_ Container dan kubelet kamu menggunakan `systemd` sebagai _driver_ cgroup
```

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+akan menstabilkan sistem. Silahkan perhatikan opsi `native.cgroupdriver=systemd` dalam pengaturan Docker di bawah ini.
+
+{{< caution >}}
+Mengubah driver cgroup dari Node yang telah bergabung kedalam sebuah Cluster sangat tidak direkomendasikan.
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:54]: use klaster instead of Cluster
```suggestion
Mengubah driver cgroup dari Node yang telah bergabung kedalam sebuah klaster sangat tidak direkomendasikan.
```

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+Jika kubelet telah membuat Pod menggunakan semantik dari sebuah _driver_ cgroup, mengubah _runtime_ Container
+ke _driver_ cgroup yang lain dapat mengakibatkan kesalahan pada saat percobaan untuk membuat kembali PodSandbox
+untuk Pod yang sudah ada. Menjalankan ulang (_restart_) kubelet mungkin tidak menyelesaikan kesalahan tersebut. Rekomendasi yang dianjurkan
+adalah untuk menguras Node dari beban kerjanya, menghapusnya dari Cluster dan menggabungkannya kembali.
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:58]: use klaster instead of Cluster
```suggestion
adalah untuk menguras Node dari beban kerjanya, menghapusnya dari klaster dan menggabungkannya kembali.
```

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 8]
```diff
@@ -0,0 +1,414 @@
...
+
+{{< /caution >}}
+
+## Docker
+
+Pada setiap mesin kamu, installah Docker.
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:64]: rewording might be better
```suggestion
Pada setiap mesin kamu, mari menginstal Docker.
```

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 9]
```diff
@@ -0,0 +1,414 @@
...
+
+Gunakan perintah berikut untuk menginstal Docker pada sistem kamu:
+
+{{< tabs name="tab-cri-docker-installation" >}}
+{{% tab name="Ubuntu 16.04+" %}}
+
+```shell
+# (Install Docker CE)
+## Set up the repository:
+### Install packages to allow apt to use a repository over HTTPS
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:76]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 10]
```diff
@@ -0,0 +1,414 @@
...
+apt-get update && apt-get install -y \
+  apt-transport-https ca-certificates curl software-properties-common gnupg2
+```
+
+```shell
+# Add Docker’s official GPG key:
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:82]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 11]
```diff
@@ -0,0 +1,414 @@
...
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+```
+
+```shell
+# Add the Docker apt repository:
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:87]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 12]
```diff
@@ -0,0 +1,414 @@
...
+add-apt-repository \
+  "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) \
+  stable"
+```
+
+```shell
+# Install Docker CE
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:95]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 13]
```diff
@@ -0,0 +1,414 @@
...
+apt-get update && apt-get install -y \
+  containerd.io=1.2.13-2 \
+  docker-ce=5:19.03.11~3-0~ubuntu-$(lsb_release -cs) \
+  docker-ce-cli=5:19.03.11~3-0~ubuntu-$(lsb_release -cs)
+```
+
+```shell
+# Set up the Docker daemon
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:103]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 14]
```diff
@@ -0,0 +1,414 @@
...
+systemctl daemon-reload
+systemctl restart docker
+```
+{{% /tab %}}
+{{% tab name="CentOS/RHEL 7.4+" %}}
+
+```shell
+# (Install Docker CE)
+## Set up the repository
+### Install required packages
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:131]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 15]
```diff
@@ -0,0 +1,414 @@
...
+yum install -y yum-utils device-mapper-persistent-data lvm2
+```
+
+```shell
+## Add the Docker repository
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:136]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 16]
```diff
@@ -0,0 +1,414 @@
...
+yum-config-manager --add-repo \
+  https://download.docker.com/linux/centos/docker-ce.repo
+```
+
+```shell
+# Install Docker CE
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:142]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 17]
```diff
@@ -0,0 +1,414 @@
...
+yum update -y && yum install -y \
+  containerd.io-1.2.13 \
+  docker-ce-19.03.11 \
+  docker-ce-cli-19.03.11
+```
+
+```shell
+## Create /etc/docker
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:150]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 18]
```diff
@@ -0,0 +1,414 @@
...
+mkdir /etc/docker
+```
+
+```shell
+# Set up the Docker daemon
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:155]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 19]
```diff
@@ -0,0 +1,414 @@
...
+}
+EOF
+```
+
+```shell
+mkdir -p /etc/systemd/system/docker.service.d
+```
+
+```shell
+# Restart Docker
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:176]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 20]
```diff
@@ -0,0 +1,414 @@
...
+systemctl daemon-reload
+systemctl restart docker
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+Jika kamu layanan Docker berjalan pada saat memulai pertama (_boot_), jalankan perintah ini:
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:183]: this doesn't make sense, maybe add helper word?
```suggestion
Jika kamu menginginkan layanan Docker berjalan dari saat memulai pertama (_boot_), jalankan perintah ini:
```

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 21]
```diff
@@ -0,0 +1,414 @@
...
+Untuk informasi lebih lanjut, lihatlah [Matriks kompatibilitas CRI-O](https://github.com/cri-o/cri-o).
+{{< /note >}}
+
+### Prasyarat
+
+```shell
+modprobe overlay
+modprobe br_netfilter
+
+# Set up required sysctl params, these persist across reboots.
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:209]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 22]
```diff
@@ -0,0 +1,414 @@
...
+EOF
+
+sysctl --system
+```
+
+{{< tabs name="tab-cri-cri-o-installation" >}}
+{{% tab name="Debian" %}}
+
+```shell
+# Debian Unstable/Sid
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:223]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 23]
```diff
@@ -0,0 +1,414 @@
...
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_Unstable/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_Unstable/Release.key -O- | sudo apt-key add -
+```
+
+```shell
+# Debian Testing
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:229]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 24]
```diff
@@ -0,0 +1,414 @@
...
+```shell
+sudo apt-get install cri-o-1.17
+```
+
+{{% /tab %}}
+
+{{% tab name="Ubuntu 18.04, 19.04 and 19.10" %}}
+
+```shell
+# Configure package repository
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:256]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 25]
```diff
@@ -0,0 +1,414 @@
...
+. /etc/os-release
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/x${NAME}_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/x${NAME}_${VERSION_ID}/Release.key -O- | sudo apt-key add -
+sudo apt-get update
+```
+
+```shell
+# Install CRI-O
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:264]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 26]
```diff
@@ -0,0 +1,414 @@
...
+sudo apt-get install cri-o-1.17
+```
+{{% /tab %}}
+
+{{% tab name="CentOS/RHEL 7.4+" %}}
+
+```shell
+# Install prerequisites
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:272]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 27]
```diff
@@ -0,0 +1,414 @@
...
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/CentOS_7/devel:kubic:libcontainers:stable.repo
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}/CentOS_7/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}.repo
+```
+
+```shell
+# Install CRI-O
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:278]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 28]
```diff
@@ -0,0 +1,414 @@
...
+```shell
+cat > /etc/modules-load.d/containerd.conf <<EOF
+overlay
+br_netfilter
+EOF
+
+modprobe overlay
+modprobe br_netfilter
+
+# Setup required sysctl params, these persist across reboots.
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:319]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 29]
```diff
@@ -0,0 +1,414 @@
...
+
+### Menginstal containerd
+
+{{< tabs name="tab-cri-containerd-installation" >}}
+{{% tab name="Ubuntu 16.04" %}}
+
+```shell
+# (Install containerd)
+## Set up the repository
+### Install packages to allow apt to use a repository over HTTPS
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:337]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 30]
```diff
@@ -0,0 +1,414 @@
...
+apt-get update && apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+```
+
+```shell
+## Add Docker’s official GPG key
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:342]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 31]
```diff
@@ -0,0 +1,414 @@
...
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+```
+
+```shell
+## Add Docker apt repository.
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:347]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 32]
```diff
@@ -0,0 +1,414 @@
...
+add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) \
+    stable"
+```
+
+```shell
+## Install containerd
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:355]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 33]
```diff
@@ -0,0 +1,414 @@
...
+apt-get update && apt-get install -y containerd.io
+```
+
+```shell
+# Configure containerd
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:360]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 34]
```diff
@@ -0,0 +1,414 @@
...
+mkdir -p /etc/containerd
+containerd config default > /etc/containerd/config.toml
+```
+
+```shell
+# Restart containerd
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:366]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 35]
```diff
@@ -0,0 +1,414 @@
...
+systemctl restart containerd
+```
+{{% /tab %}}
+{{% tab name="CentOS/RHEL 7.4+" %}}
+
+```shell
+# (Install containerd)
+## Set up the repository
+### Install required packages
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:375]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 36]
```diff
@@ -0,0 +1,414 @@
...
+yum install -y yum-utils device-mapper-persistent-data lvm2
+```
+
+```shell
+## Add docker repository
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:380]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 37]
```diff
@@ -0,0 +1,414 @@
...
+yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+```
+
+```shell
+## Install containerd
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:387]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 38]
```diff
@@ -0,0 +1,414 @@
...
+yum update -y && yum install -y containerd.io
+```
+
+```shell
+## Configure containerd
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:392]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md [hunk 39]
```diff
@@ -0,0 +1,414 @@
...
+mkdir -p /etc/containerd
+containerd config default > /etc/containerd/config.toml
+```
+
+```shell
+# Restart containerd
```
Comment (girikuncoro) [inline content/id/docs/setup/production-environment/container-runtimes.md:398]: you can translate the comment as well

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 2 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md:38]: I prefer `cgroup` because it is a name.
https://en.wikipedia.org/wiki/Cgroups

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 6 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md:54]: Cluster is not API name?
https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cluster

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 7 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md:58]: Cluster is not API name?
https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cluster

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 1 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: fixed in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 3 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: fixed in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 4 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: fixed in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 5 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: fixed in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 8 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: fixed in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 9 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 10 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 11 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 12 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 13 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 14 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 15 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 16 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 17 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 18 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 19 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 20 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: FIxed in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 21 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 22 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md:223]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 23 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md:229]: This name of the version, no need to translate!

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 24 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 25 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 26 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 27 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 29 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 30 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 31 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 32 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 33 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 34 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 35 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 36 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 37 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 38 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 39 above)
Comment (ariscahyadi) [reply content/id/docs/setup/production-environment/container-runtimes.md]: Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 2 above)
Comment (girikuncoro) [reply content/id/docs/setup/production-environment/container-runtimes.md:38]: cool let's rephrase then?
```suggestion
dan menggunakan _control group_ (`cgroup`) root  dan proses ini akan bertindak sebagai manajer cgroup. Systemd memiliki integrasi yang ketat
```

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 7 above)
Comment (girikuncoro) [reply content/id/docs/setup/production-environment/container-runtimes.md:58]: Cluster is not native API name, you should refer to the API reference instead of that glossary
API reference: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 23 above)
Comment (girikuncoro) [reply content/id/docs/setup/production-environment/container-runtimes.md:229]: oh my bad it's a version of debain 😅 I thought it can be translated to something like "Menguji Debian"

File: content/id/docs/setup/production-environment/container-runtimes.md (hunk 6 above)
Comment (girikuncoro) [reply content/id/docs/setup/production-environment/container-runtimes.md:54]: Cluster is not native API name, you should refer to the API reference instead of that glossary
API reference: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/

**Comment by girikuncoro** [comment]
@ariscahyadi all looking good apart from "Cluster" needs to be renamed to "klaster" and the nit _control group_. Feel free to do this as follow up PR, thanks for your contribution!

/lgtm
/approve
